From e418677e7d09b6a44805cbca177a446016eea23f Mon Sep 17 00:00:00 2001
From: Patryk <patryk@lizardfs.com>
Date: Sun, 23 Aug 2020 09:52:26 +0200
Subject: [PATCH] build: Fix compilation for gcc-10

There was a lacking include for cstdio in daemonize.cc.
This header is no longer included implicitly since gcc-10,
hence the compilation error.
See: https://github.com/CopernicaMarketingSoftware/PHP-CPP/pull/440

Change-Id: I5b22d4759e100e270aa0732d32407b0edc16f500
---
 src/mount/fuse/daemonize.cc | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/mount/fuse/daemonize.cc b/src/mount/fuse/daemonize.cc
index 8d0ccf1c4..7ba3eeadc 100644
--- a/src/mount/fuse/daemonize.cc
+++ b/src/mount/fuse/daemonize.cc
@@ -19,9 +19,10 @@
 #include "common/platform.h"
 #include "daemonize.h"
 
-#include <errno.h>
+#include <cerrno>
+#include <cstdio>
+#include <cstring>
 #include <fcntl.h>
-#include <string.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
